mlcomp/parallelm/common/bg_actor.py

```diff
@@ -18,13 +18,13 @@ def __init__(self, mlops, ml_engine, polling_interval_sec=10.0):
         self._polling_interval_sec = polling_interval_sec
 
         self._condition = threading.Condition()
-        self._stop = False
+        self._stop_flag = False
```

Comment:
I would change it to '_stop_gracefully' or '_exit'

Reply:
done